<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherResourcesClass</class>
 <widget class="QMainWindow" name="TeacherResourcesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeacherResources</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="backgorund">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-50</y>
      <width>741</width>
      <height>522</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../OneDrive - ПГКПИ - Бургас/Проектни дейности/sprint-eschool-dataforce/DataForce/PL/images/studentResourses.jpg</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="navbar">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 177, 210);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>5</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(169, 213, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Log out </string>
     </property>
    </widget>
    <widget class="QLabel" name="menuButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="testsButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Tests</string>
     </property>
    </widget>
    <widget class="QLabel" name="resourcesButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Resources</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addResourceButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>432</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(169, 213, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteResourceButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>432</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(169, 213, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openResourceButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>432</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(169, 213, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QListWidget" name="pdfListWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>54</y>
      <width>711</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
